nomos-da/network/core/src/protocols/replication/behaviour.rs

```diff
@@ -289,10 +310,10 @@ where
     /// sent again.
     fn send_message_impl(&mut self, waker: Option<&Waker>, message: &DaMessage) {
         let message_id = (message.share.blob_id.to_vec(), message.subnetwork_id);
```

Comment:
Oh, so it is actually the blob_id+subnetwork. Makes sense. But don't we have a `BlobId` type somewhere to be used in the declaration above?